pkg/apis/duck/v1/eventpoliciesstatus_type.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2024 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+// AppliedEventPoliciesStatus contains the list of policies which apply to a resource.
+// This type is intended to be embedded into a status struct.
+type AppliedEventPoliciesStatus struct {
+	// Policies holds the list of applied EventPolicies
+	// +optional
+	Policies []AppliedEventPoliciesStatusPolicy `json:"policies,omitempty"`
+}
+
+// AppliedEventPoliciesStatusPolicy is the reference to a EventPolicy
+type AppliedEventPoliciesStatusPolicy struct {
```

Comment:
this name is very difficult to read, maybe:
- AppliedEventPolicyRef
- AppliedEventPolicy
- something else?

Reply:
:+1: Updated